model/src/main/scala/cromwell/pipeline/model/wrapper/RunId.scala

```diff
@@ -19,6 +19,11 @@ object RunId extends Wrapped.Companion {
     value,
     NonEmptyChain.one("Invalid RunId")
   )
+  def fromString(value: String): ValidationResult[Wrapper] =
```

Comment:
Why do we need this method? Can't we just use `.from` ?

Reply:
Yep. You are right! I`ll change it.

Reply:
Done.